examples/gopaste/snapcraft.yaml

```diff
@@ -9,16 +9,8 @@ apps:
     command: bin/gopaste
     daemon: simple
     plugs:
-      - old-security
-
-plugs:
-  old-security:
-    caps:
-      - network-listener
-      - network-service
-      - network-management
```

Comment:
Technically this would need network-control but please don't add that. Live with the denial instead until https://launchpad.net/bugs/1465724 is fixed.